ompi/datatype/ompi_datatype_create_darray.c

```diff
@@ -216,9 +216,14 @@ int32_t ompi_datatype_create_darray(int size,
     }
 
     /* Build up array */
+    displs[0] = st_offsets[start_loop];
```

Comment:
did you really mean to commit changes to this file ?
before entering the loop, the ```st_offsets``` array is uninitialized, and this commit causes a failure in the ```datatype/darray-pack``` test from the ```ibm``` test suite